fedn/cli/run_cmd.py

```diff
@@ -49,10 +54,11 @@ def reducer_cmd(ctx, discoverhost, discoverport, token, name):
 @click.option('-h', '--hostname', required=True)
 @click.option('-i', '--port', required=True)
 @click.option('-s', '--secure', required=False, default=True)
+@click.option('-c', '--max_clients', required=False,default=8)
 @click.pass_context
-def combiner_cmd(ctx, discoverhost, discoverport, token, name, hostname, port, secure):
+def combiner_cmd(ctx, discoverhost, discoverport, token, name, hostname, port, secure,max_clients):
```

Comment:
I think this should be moved to a combiner config.yaml in due time where strategy parameters could be offset from default.

Reply:
Agree. Lets create a ticket for it.